Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -805,7 +875,10 @@ def tc_get_indicator(indicator, owners, rating_threshold, confidence_threshold,
         if 'indicator_observations' in raw_indicators[0]:
             indicator_observations = raw_indicators[0]['indicator_observations']
 
-    return ec, indicators, raw_indicators, indicators_associations, indicator_groups, indicator_observations, indicator_tags
+        if 'indicator_attributes' in raw_indicators[0]:
+            indicator_attributes = raw_indicators[0]['indicator_attributes']
+
+    return ec, indicators, raw_indicators, indicators_associations, indicator_groups, indicator_observations, indicator_tags, indicator_attributes
```

Comment:
Please note lint fails on `line too long` here

Reply:
Is there a linter config that I can use?